compiler/noirc_evaluator/src/ssa/ir/instruction.rs

```diff
@@ -847,9 +847,16 @@ fn eval_constant_binary_op(
             if matches!(operator, BinaryOp::Div | BinaryOp::Mod) && rhs == 0 {
                 return None;
             }
-
             let result = function(lhs, rhs);
```

Comment:
```suggestion
            let result = function(lhs, rhs)?;
```

We can `?` here to return `None` values